addons/website/static/src/snippets/s_table_of_content/options.js

```diff
@@ -42,8 +42,8 @@ options.registry.TableOfContent = options.Class.extend({
         _.each($headings, el => {
             const $el = $(el);
             const id = 'table_of_content_heading_' + _.now() + '_' + _.uniqueId();
-            $('<a>').attr('href', "#" + id)
-                    .addClass('table_of_content_link list-group-item list-group-item-action py-2 border-0 rounded-0')
+            $('<a>').attr('href', `#${id}`)
+                    .addClass('table_of_content_link list-group-item list-group-item-action py-2 border-0 rounded-0 o_not_editable')
```

Comment:
Maybe this part and scrollspy stuff should be done as a fix (the whole commit except the new "Add content" UI)?